src/components/file-input/file-input.mdx

```diff
@@ -131,21 +116,18 @@ The `accept` prop is passed directly to the underlying HTML `<input type="file">
 Please see the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/accept) for more details on this attribute. Note that it does not prevent the user
 from uploading a file of the incorrect type, for example via drag and drop - so you might still want to use the `onChange` prop to add an error message as soon as an
 inappropriate file is added. And of course nothing takes away the need for validation on the server side. However the `accept` prop is still recommended to guide the user
+
```

Comment:
```suggestion
```

**suggestion**: I'm not sure why you added a blank line here (slip of the fingers?), but it's messing up the markdown so that the below `-` is interpreted as being a list item, which it wasn't intended to be, it's just a way of breaking up the sentence.

Reply:
Nope that's prettier doing it's thing. I've moved the hyphen to the previous line and that prevents the line break